0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -24,6 +24,7 @@ SPSP provides for exchanging basic server details needed by a client to set up a
 * **SPSP Server** - The application that handles incoming SPSP requests from the SPSP Client.
 * **SPSP Endpoint** - The specific HTTPS endpoint on the SPSP Server used for setting up a payment.
 * **STREAM Module** - Software included in the SPSP Client and Server that implements the [STREAM](../0029-stream/0029-stream.md) protocol.
+* **STREAM Receipt** - Proof provided by the payment recipient of the total amount received on a stream.
```

Comment:
Overall, these changes are looking good. One thing that I think would be really helpful (as a supplement to the changes proposed in this PR) is to _also_ create a new IL-RFC for **Stream Receipts**. That RFC should outline most of the information and flows that @sharafian [detailed in this post](https://github.com/interledger/rfcs/issues/568).

While the formalizations presented in this PR seem to support the minimal spec changes to enable Stream Receipts, I think having a formalization of how receipts are generated, used, verified, etc as per #568 is probably a good idea. 

Unless there's some particular reason not to be more specific than the changes here?

Reply:
This is addressed - thanks!